crates/crates_io_database/src/models/token.rs

```diff
@@ -80,7 +77,7 @@ impl ApiToken {
             .transaction(|conn| {
                 async move {
                     diesel::update(tokens)
-                        .set(api_tokens::last_used_at.eq(now.nullable()))
+                        .set(api_tokens::last_used_at.eq(now.into_sql::<Timestamptz>().nullable()))
```

Comment:
this is unfortunately necessary due to a quirk in diesel. see https://github.com/diesel-rs/diesel/issues/1514#issuecomment-613340122.